docs/products/m3db.md

```diff
@@ -8,15 +8,12 @@ Aiven for M3 is a fully managed **distributed time series database**, deployable
 Aiven for M3 consists of `n` number of **M3DB** and **M3 Coordinator**
 pairs (where `n` is the number of nodes as chosen for your Aiven plan).
 
-M3 is a great choice if your organization has a very large volume of
-metrics to handle, and it can be used as part of your observability
-solution. It is optimized for storing and serving time series through
-associated pairs of times and values. It also provides a reverse index
-of time series.
-
-:::note
-Aiven offers M3 because we ourselves needed a solution that would work
-with the size of our own metrics.
+:::important
+Aiven for M3DB will be deprecated later this year. Consider using
```

Comment:
I think we might need a stronger wording here to prevent customer from running M3DB. But other than that it looks good

Reply:
Update this note, let me know if this is better? or do you have any other suggestion here?